spec/serializers/rest/account_serializer_spec.rb

```diff
@@ -44,4 +45,44 @@
       expect(subject['memorial']).to be true
     end
   end
+
+  context 'when created_at is populated' do
+    before do
+      account.account_stat.update!(created_at: default_datetime)
+    end
+
+    it 'parses as RFC 3339 datetime' do
+      expect { DateTime.rfc3339(subject['created_at']) }.to_not raise_error
+    end
+  end
+
+  context 'when last_status_at is populated' do
+    before do
+      account.account_stat.update!(last_status_at: default_datetime)
+    end
+
+    it 'is serialized as yyyy-mm-dd' do
+      expect(subject['last_status_at']).to eq('2024-11-28')
+    end
+  end
+
+  context 'when mute_expires_at is populated', pending: 'creating a muted account' do
```

Comment:
Is this one actually re: something in the `AccountSerializer` ... or should be in separate `MutedAccountSerializer` spec?

In either case, something like...

`Fabricate :mute, account: x, target_account: y, mute_expires_at: 2.days.ago` - where `x` is instance of the account that created the mute and `y` is instance of account it wants to mute, would create the background condition here.

Reply:
Couldn't get that to work, so used the account's follow! and mute! methods.